### PR TITLE
Adds bee plushies

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

Adds ~~beegirls~~ beeplushies. They will be available as botany heirlooms.

## Why It's Good For The Game

They are pretty cute.
beeplushie

## Changelog
:cl:
add: Botanists can now get beeplushies as an heirloom.
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
This isn't a goon sprite is it? :eyes:

**Comment by Shapsy** [comment]
I thought it was gonna be a beepsky plush :/

**Comment by nemvar** [comment]
I made the sprite myself but it's based off http://www.squishable.com/mm5/graphics/00000001/opensquish_bumblebee_34497.jpg

**Comment by shellspeed1** [comment]
I am still unsure if fetish content or not.

**Review by ninjanomnom — DISMISSED** [review]
I rate this pr :bee: / :bee:

**Comment by spessbro** [comment]
speedmerge that shit

**Comment by 81Denton** [comment]
hip

**Comment by BeloneX** [comment]
beestation rise up

**Comment by Shadowflame909** [comment]
with all the bee related prs, all signs point to bee girls

**Comment by Rowelll** [comment]
indeed

**Comment by Manly-Dwarf** [comment]
all hail our goonstation overlords

**Comment by kopoba** [comment]
Bee

File: code/datums/outputs.dm [hunk 1]
```diff
@@ -111,4 +111,8 @@
 
 /datum/outputs/squelch
 	text = "You hear a horrendous squelching sound."
-	sounds = 'sound/effects/blobattack.ogg'
\ No newline at end of file
+	sounds = 'sound/effects/blobattack.ogg'
+
+/datum/outputs/bee
+	text = "You hear a buzzing noise."
+	sounds = 'sound/voice/moth/scream_moth.ogg'
```
Comment (ShizCalev) [inline code/datums/outputs.dm]: New line at the end please

File: code/game/objects/items/plushes.dm [hunk 2]
```diff
@@ -523,4 +523,13 @@
...
+	AddComponent(/datum/component/edit_complainer)
+
+/obj/item/toy/plush/beeplushie
+	name = "bee plushie"
+	desc = "A cute toy that resembles an even cuter bee."
+	icon_state = "plushie_h"
+	item_state = "plushie_h"
+	attack_verb = list("stung")
+	gender = FEMALE
+	component = /datum/component/squeak/beeplushie
```
Comment (ShizCalev) [inline code/game/objects/items/plushes.dm]: ditto

**Pull request merged**